docs/src/main/sphinx/connector/hudi.rst

```diff
@@ -157,3 +157,25 @@ Here are some sample queries:
     ------------+--------+
      2018-08-31 |  99  |
     (1 rows)
+
+Time travel queries
+^^^^^^^^^^^^^^^^^^^
+
+The connector offers the ability to query historical data.
+This allows you to query the table as it was when a previous snapshot
+of the table was taken, even if the data has since been modified or deleted.
+
+The historical data of the table can be retrieved by specifying the
+snapshot identifier corresponding to the version of the table that
+needs to be retrieved::
+
+   SELECT *
+   FROM hudi.testdb.customer_orders FOR VERSION AS OF 8954597067493422955
+
+A different approach of retrieving historical data is to specify
+a point in time in the past, such as a day or week ago. The latest snapshot
+of the table taken before or at the specified timestamp in the query is
+internally used for providing the previous state of the table::
+
+   SELECT *
+   FROM hudi.testdb.customer_orders FOR TIMESTAMP AS OF TIMESTAMP '2022-03-23 09:59:29.803 Europe/Vienna'
```

Comment:
Please use "example" as the catalog name.

Reply:
Also, could you document how users can find those versions (`8954597067493422955` and `'2022-03-23 09:59:29.803 Europe/Vienna'`) from the existing tables?

Reply:
added the description in hudi.rst. :)